tuna/connection.py

```diff
@@ -190,87 +274,49 @@ def exec_command(self, cmd, timeout=SSH_TIMEOUT, abort=None, proc_line=None):
       if not proc_line:
         # pylint: disable-next=unnecessary-lambda-assignment ; more readable
         proc_line = lambda x: self.logger.info(line.strip())
-      ret_out = StringIO()
-      ret_err = e_var
+      ret_out: Optional[TextIO] = StringIO()
+      ret_err: ChannelStderrFile = e_var
```

Comment:
This type would be a union with StringIO, though ideally they would both be converted to StringIO.

Reply:
I tried this option, it iwould not resolve